docs/src/content/docs/fr/guides/components.mdx

```diff
@@ -205,3 +205,49 @@ Le code ci-dessus génère ce qui suit sur la page :
 Une liste de toutes les icônes disponibles est affichée ci-dessous avec leurs noms associés. Cliquez sur une icône pour copier le code du composant.
 
 <IconsList />
+
+### Code
+
+Utilisez le composant `<Code>` pour rendre le code en évidence syntaxique lorsque l'utilisation d'un [bloc de code Markdown](/fr/guides/authoring-content/#blocs-de-code) n'est pas possible, par exemple, pour rendre des données provenant de sources externes comme des fichiers, des bases de données ou des API.
+
+Voir la documentation [Expressive Code "Code Component"](https://expressive-code.com/key-features/code-component/) pour plus de détails sur les fonctions que `<Code>` supporte.
+
+```mdx
+# src/content/docs/example.mdx
```

Comment:
```suggestion
# src/content/docs/exemple.mdx
```

We usually try to translate file name when this make sense, e.g. the first example on [this page](https://starlight.astro.build/fr/reference/frontmatter/).

Reply:
okay! I didn't find it necessary to translate even the example files... in the examples. But if I have to, I'll merge the whole thing! Thanks!